src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -338,15 +361,26 @@ public SyntaxNode EventDeclaration(IEventSymbol symbol)
             DeclarationModifiers modifiers = default(DeclarationModifiers),
             IEnumerable<SyntaxNode> parameters = null,
             IEnumerable<SyntaxNode> addAccessorStatements = null,
-            IEnumerable<SyntaxNode> removeAccessorStatements = null);
+            IEnumerable<SyntaxNode> removeAccessorStatements = null,
+            IEnumerable<SyntaxNode> raiseAccessorStatements = null);
+
+        // 2.0 BACKCOMPAT OVERLOAD -- DO NOT TOUCH
+        public SyntaxNode CustomEventDeclaration(
+            IEventSymbol symbol,
+            IEnumerable<SyntaxNode> addAccessorStatements,
+            IEnumerable<SyntaxNode> removeAccessorStatements)
+        {
+            return CustomEventDeclarationFromSymbol(symbol, addAccessorStatements, removeAccessorStatements);
+        }
 
         /// <summary>
         /// Creates a custom event declaration from an existing event symbol.
         /// </summary>
-        public SyntaxNode CustomEventDeclaration(
+        public SyntaxNode CustomEventDeclarationFromSymbol(
             IEventSymbol symbol,
             IEnumerable<SyntaxNode> addAccessorStatements = null,
-            IEnumerable<SyntaxNode> removeAccessorStatements = null)
+            IEnumerable<SyntaxNode> removeAccessorStatements = null,
+            IEnumerable<SyntaxNode> raiseAccessorStatements = null)
```

Comment:
>IEnumerable<SyntaxNode> raiseAccessorStatements = null) [](start = 12, length = 55)

The reason this was not here in the existing API because only VB has raise events.  It it is not possible to write language agnostic code here to build raise events without knowing you are building VB only.